extension/src/popup/components/account/AccountAssets/styles.scss

```diff
@@ -44,6 +44,8 @@ $loader-light-color: #444961;
     }
 
     &--error {
+      width: 32px;
```

Comment:
Unrelated but I snuck these in here since these 10liners fix the loading state for assets on account details.